drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -2132,7 +2132,7 @@ void RasterizerSceneGLES3::_render_shadow_pass(RID p_light, RID p_shadow_atlas,
 				light_transform = light_storage->light_instance_get_shadow_transform(p_light, p_pass);
 				shadow_size = shadow_size / 2;
 			} else {
-				ERR_FAIL_MSG("Dual paraboloid shadow mode not supported in GL Compatibility renderer. Please use Cubemap shadow mode instead.");
+				ERR_FAIL_MSG("Dual paraboloid shadow mode not supported in Compatibility renderer. Please use Cubemap shadow mode instead.");
```

Comment:
While you're here, maybe?
```suggestion
				ERR_FAIL_MSG("Dual paraboloid shadow mode not supported in the Compatibility renderer. Please use CubeMap shadow mode instead.");
```
Unsure if the rest of the messages herere are worded similarly, though.

Reply:
That's just an oversight. My guidelines were to prefer the fully spelled out "the Compatibility renderer" but allow the briefer "in Compatibility", "Compatibility doesn't support", etc sometimes.